README.md

```diff
@@ -6,7 +6,7 @@
 
 🦀 Rust good C bad.
 
-### [Amazing poem I wrote.](amazing_poem.md)
+### [Amazing poem I wrote.](https://github.com/fipachu/PythonProblems/blob/e085b8e31d80db21f769c0fd1e843cb06c7fa95d/Topics/Class%20vs%20instance/Sphere/main.py)
```

Comment:
NOTE: this is a link to the commit in which the source code was added to PythonProblems, not to the main branch.